auth/proxy/firebaseJwt.ts

```diff
@@ -62,13 +63,14 @@ const getSigningKey = async (kid: string): Promise<string> => {
 
 const isKnownApp = (
     payload: JwtPayload,
-    firebaseAppIds: string[],
+    configValues: AppConfig,
```

Comment:
Could we leave this as is and pass in: `[configValues.firebaseIosAppId, configValues.firebaseAndroidAppId]` instead?